x-pack/plugins/canvas/canvas_plugin_src/uis/arguments/image_upload/index.js

```diff
@@ -123,18 +124,19 @@ class ImageUpload extends React.Component {
     }
 
     const pasteImageUrl = urlTypeSrc ? (
-      <form onSubmit={this.setSrcUrl} className="eui-textRight">
+      <EuiForm onSubmit={this.setSrcUrl} className="eui-textRight">
         <EuiFieldText
           compressed
-          value={this.state.url}
+          defaultValue={this.state.url}
```

Comment:
We just had a lot of issues with uncontrolled forms, so my initial reaction here is fear of bringing them back ;).

I know you touched on this in the description, but I don't really understand why this needs to be uncontrolled. Can you clarify why?

Reply:
There was a `props` warning in the code I was testing, so I decided to fix it as was recommended.  I can revert it, but it seems to me this is a pretty bad code smell issue.